streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -312,79 +312,6 @@ public void testDrivingSimpleTopologyWithDroppingPartitioner() {
         assertTrue(outputTopic1.isEmpty());
     }
 
-    @Test
-    public void testDrivingStatefulTopology() {
```

Comment:
Why are we removing this test? Seems we should keep it. It's not marked as `@Deprecated` and does not say "old API" in it's name either, nor is their a comment that it would test old API.

Reply:
@pegasas -- i don't see any reply on this comment (same for the other 3 below)